docs/index.html

```diff
@@ -571,25 +571,25 @@ <h3 class="heading settled" data-level="4.3" id="storage"><span class="secno">4.
+   <p><a data-link-type="dfn" href="#author" id="ref-for-author">Authors</a> and Issuers have an interest in versioning to track advancements and changes over time both for formatting changes (e.g. supporting <a data-link-type="biblio" href="#biblio-json-schema" title="JSON Schema: A Media Type for Describing JSON Documents">[JSON-SCHEMA]</a> <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00">draft-bhutton-json-schema-00</a> as opposed to <a href="https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-01">Draft 7</a>) as well as field-level changes (e.g. adding a new required field) as a schema evolves over time. Holders have an interest in versioning to gain an understanding in where their credentials can be used as they may receive requests for <a data-link-type="dfn" href="#presentation" id="ref-for-presentation">presentation</a> filtering by schema ID. Similarly, Verifiers have an interest in versioning to know which data, or schema versions they should accept and known how to process in their systems.</p>
    <p>Gudelines for versioning can be found in the <a href="#versioning_guidelines">Versioning Guidelines section of this document</a>.</p>
    <h3 class="heading settled" data-level="4.5" id="processing"><span class="secno">4.5. </span><span class="content">Processing</span><a class="self-link" href="#processing"></a></h3>
-   <p>There is wide support for JSON Schema processing and <a data-link-type="biblio" href="#biblio-json-schema-implmentations">[JSON-SCHEMA-IMPLMENTATIONS]</a> can be found in many programming languages.</p>
+   <p>There is wide support for JSON Schema processing and <a data-link-type="biblio" href="#biblio-json-schema-implmentations" title="JSON Schema Implementations">[JSON-SCHEMA-IMPLMENTATIONS]</a> can be found in many programming languages.</p>
    <p>To process a <a data-link-type="dfn" href="#verifiable-credential" id="ref-for-verifiable-credential⑦">Verifiable Credential</a> against a <a data-link-type="dfn" href="#credential-schema" id="ref-for-credential-schema⑧">Credential Schema</a> one must first identify which sections of credential the schema applies to, as specified in the <code>credentialSchema</code> property of the credential, and next apply <a href="#processing_validation">validation</a>.</p>
```

Comment:
`id="ref-for-credential-schema⑧"`? `⑧`?

Reply:
I think this file should be git ignored probably, its compiled from bikeshed.